src/adv.py

```diff
@@ -67,33 +67,55 @@
 #
 # If the user enters "q", quit the game.
 
-user_input = None
+choice = None
+moved = True
+
+while choice != 'q':
+    if (choice in ['n', 's', 'w', 'e', None] and moved):
+        print("===============\n")
+        print(f"{player.name} enters \"{player.room.name}\"")
+        print(f"{player.room.desc}\n")
 
-while user_input is not 'q':
-    print("===============\n")
-    print(f"{player.name} enters \"{player.room.name}\"")
-    print(f"{player.room.desc}\n")
     player.room.print_items()
-    user_input = input(
-        "\nMove (n, s, w, e) or Take Action (get/take/drop/inventory): ")
 
-# Movement Logic
-    if len(user_input.split(' ')) == 1:
-        if (user_input == 'n' or user_input == 'N') and hasattr(player.room, 'n_to'):
+# Player input
+    choice = input(
+        "\nCheck Inventory (i/inv), Move (n, s, w, e) or Take Action (get/take/drop): ")
+    choice_arr = choice.split(' ')
+    choice_len = len(choice_arr)
+# Movement and Player Inventory Logic (expects 1 argument)
+    if choice_len == 1:
+        if (choice in ['n', 'N']) and hasattr(player.room, 'n_to'):
```

Comment:
I think you're the first one I see using "in [array]". very nice, that'll be really useful in Python.